### DISCUSSION TITLE
Found A new one uninterrupted brower bypass (WORKS ON ALL VERSIONS)

### DISCUSSION THREAD

**Discussion by Thecodeisrereall** [3kh0/ext-remover#1003] [Webview JOIN TITANIUM NETWORK TO SHARE PLEASE]
Found a basically proxy using the github exploit
using geogebra installed as an app on my schools computer you can sign in and in the app and click the logo in the sign in page to get to the main page of the actual website where you can log in with microsoft accound and go to github (IMPORTANT) press reset password and then click the cat.
![Screenshot 2024-04-10 11 55 08 AM](https://github.com/3kh0/ext-remover/assets/85471742/27413f27-3402-4df1-857a-0a40e13b4981)
![Screenshot 2024-04-10 11 55 00 AM](https://github.com/3kh0/ext-remover/assets/85471742/56ba1e09-4562-42bd-a8e5-d88ca72afd0f)

**Comment by myschoolisass** [comment]
this was a webview exploit already found lol

**Reply by Thecodeisrereall** [reply]
well yea but this goes around that

**Reply by Thecodeisrereall** [reply]
[Screen recording 2024-04-10 12.06.00 PM.webm](https://github.com/3kh0/ext-remover/assets/85471742/dc801825-c4b4-4c59-a25b-82da57e67553)
this is how todo it

**Comment by py660** [comment]
this applies to **all** apps which allow google, github, or microsoft sign in.

**Comment by Blobby-Boi** [comment]
It also works with many other apps such as S2Bot 4 Scratch
